--- /usr/include/execinfo.h.old 2018-03-14 21:16:29.000000000 +0000
+++ /usr/include/execinfo.h     2018-12-28 03:21:33.257248216 +0000
@@ -29,16 +29,8 @@
 #ifndef _EXECINFO_H_
 #define _EXECINFO_H_
 
-#ifdef __cplusplus
-extern "C" {
-#endif
-
-int     backtrace(void **, int);
-char ** backtrace_symbols(void *const *, int);
-void    backtrace_symbols_fd(void *const *, int, int);
-
-#ifdef __cplusplus
-}
-#endif
+#define backtrace(...) (1)
+#define backtrace_symbols(...) 0
+#define backtrace_symbols_fd(...) ((void) 0)
 
 #endif /* _EXECINFO_H_ */