diff --git a/kf5-frameworks-build-include b/kf5-frameworks-build-include
index 67707df..c253039 100644
--- a/kf5-frameworks-build-include
+++ b/kf5-frameworks-build-include
@@ -54,7 +54,7 @@ module-set frameworks
     # if you use Make as build system add to cmake-options -G "CodeBlocks - Unix Makefiles"
     # if you use Ninja as build system add to cmake-options -G "CodeBlocks - Ninja"
     # see http://www.cmake.org/cmake/help/v3.0/generator/CodeBlocks.html
-    cmake-options -DBUILD_TESTING=TRUE
+#    cmake-options -DBUILD_TESTING=TRUE
     #tag v5.50.0-rc1
 end module-set
 
