*** debian/rules.bak	Tue Dec 11 23:43:34 2018
--- debian/rules	Tue Dec 11 23:49:44 2018
***************
*** 105,111 ****
    endif
  
  	confflags_DIRECT_RENDERING = --enable-driglx-direct
! 	confflags_GBM = --enable-gbm
  	confflags_GALLIUM += --enable-vdpau
  	confflags_GALLIUM += --enable-va
  	confflags_GALLIUM += --enable-gallium-extra-hud
--- 105,111 ----
    endif
  
  	confflags_DIRECT_RENDERING = --enable-driglx-direct
! 	confflags_GBM = --disable-gbm
  	confflags_GALLIUM += --enable-vdpau
  	confflags_GALLIUM += --enable-va
  	confflags_GALLIUM += --enable-gallium-extra-hud
***************
*** 123,143 ****
  confflags_GALLIUM += --with-gallium-drivers="$(GALLIUM_DRIVERS)"
  
  confflags += \
! 	--enable-dri \
  	--with-dri-drivers="$(DRI_DRIVERS)" \
  	--with-dri-driverdir=/usr/lib/$(DEB_HOST_MULTIARCH)/dri \
  	--with-dri-searchpath='/usr/lib/$(DEB_HOST_MULTIARCH)/dri:\$$$${ORIGIN}/dri:/usr/lib/dri' \
  	--with-vulkan-drivers="$(VULKAN_DRIVERS)" \
  	--enable-osmesa \
  	--enable-glx-tls \
- 	--enable-shared-glapi \
  	--enable-texture-float \
  	--disable-xvmc \
  	--disable-omx \
  	$(confflags_DIRECT_RENDERING) \
  	$(confflags_GBM) \
  	$(confflags_DRI3) \
- 	$(confflags_EGL) \
  	$(confflags_GALLIUM) \
  	$(confflags_GLES) \
  	$(buildflags)
--- 123,146 ----
  confflags_GALLIUM += --with-gallium-drivers="$(GALLIUM_DRIVERS)"
  
  confflags += \
! 	--disable-dri \
  	--with-dri-drivers="$(DRI_DRIVERS)" \
  	--with-dri-driverdir=/usr/lib/$(DEB_HOST_MULTIARCH)/dri \
  	--with-dri-searchpath='/usr/lib/$(DEB_HOST_MULTIARCH)/dri:\$$$${ORIGIN}/dri:/usr/lib/dri' \
  	--with-vulkan-drivers="$(VULKAN_DRIVERS)" \
  	--enable-osmesa \
  	--enable-glx-tls \
  	--enable-texture-float \
  	--disable-xvmc \
  	--disable-omx \
+ 	--enable-static \
+ 	--disable-shared \
+ 	--enable-llvm \
+ 	--disable-gbm \
+ 	--disable-egl \
  	$(confflags_DIRECT_RENDERING) \
  	$(confflags_GBM) \
  	$(confflags_DRI3) \
  	$(confflags_GALLIUM) \
  	$(confflags_GLES) \
  	$(buildflags)
