diff --git a/srcpkgs/dbus/template b/srcpkgs/dbus/template
index a4fb81ac06..ab04a8810c 100644
--- a/srcpkgs/dbus/template
+++ b/srcpkgs/dbus/template
@@ -4,7 +4,7 @@ version=1.12.12
 revision=1
 build_style=gnu-configure
 configure_args="--disable-selinux --enable-inotify --with-dbus-user=dbus
- --enable-xml-docs --disable-static --disable-tests --enable-epoll
+ --enable-xml-docs --enable-static --disable-tests --enable-epoll
  --disable-asserts --disable-systemd --disable-user-session
  --with-system-socket=/run/dbus/system_bus_socket --disable-doxygen-docs
  --with-system-pid-file=/run/dbus/pid --with-console-auth-dir=/run/console"
@@ -34,6 +34,7 @@ dbus-devel_package() {
 		vmove "usr/lib/dbus-*"
 		vmove usr/lib/pkgconfig
 		vmove "usr/lib/*.so"
+		vmove "usr/lib/*.a"
 		vmove usr/share/doc
 		vmove usr/lib/cmake
 	}
diff --git a/srcpkgs/fontconfig/template b/srcpkgs/fontconfig/template
index 1deb07eca6..18f172e819 100644
--- a/srcpkgs/fontconfig/template
+++ b/srcpkgs/fontconfig/template
@@ -3,7 +3,7 @@ pkgname=fontconfig
 version=2.13.1
 revision=1
 build_style=gnu-configure
-configure_args="--disable-static --enable-docs --with-cache-dir=/var/cache/${pkgname}"
+configure_args="--enable-static --enable-docs --with-cache-dir=/var/cache/${pkgname}"
 hostmakedepends="gperf pkg-config"
 makedepends="expat-devel freetype-devel libuuid-devel"
 conf_files="/etc/fonts/fonts.conf"
@@ -35,6 +35,7 @@ fontconfig-devel_package() {
 		vmove usr/lib/pkgconfig
 		vmove usr/share/man/man3
 		vmove "usr/lib/*.so"
+		vmove "usr/lib/*.a"
 		vmove usr/share/gettext
 	}
 }
diff --git a/srcpkgs/icu/template b/srcpkgs/icu/template
index 3043b531d4..9a386be3e2 100644
--- a/srcpkgs/icu/template
+++ b/srcpkgs/icu/template
@@ -5,6 +5,7 @@ revision=2
 wrksrc=icu
 build_wrksrc=source
 build_style=gnu-configure
+configure_args="--enable-static"
 hostmakedepends="pkg-config"
 short_desc="Robust and full-featured Unicode services"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
@@ -66,6 +67,7 @@ icu-devel_package() {
 		vmove usr/bin/icu-config
 		vmove "usr/share/man/man1/icu-config*"
 		vmove "usr/lib/*.so"
+		vmove "usr/lib/*.a"
 	}
 }
 icu-libs_package() {
diff --git a/srcpkgs/libGL/patches/glapi-static.patch b/srcpkgs/libGL/patches/glapi-static.patch
new file mode 100644
index 0000000000..7bcddc6a04
--- /dev/null
+++ b/srcpkgs/libGL/patches/glapi-static.patch
@@ -0,0 +1,21 @@
+--- src/mapi/glapi/meson.build.orig
++++ src/mapi/glapi/meson.build
+@@ -76,6 +76,7 @@
+   c_args : [c_msvc_compat_args, static_glapi_args],
+   dependencies : [dep_thread, dep_selinux],
+   build_by_default : false,
++  install : true,
+ )
+ 
+ if with_any_opengl and not with_shared_glapi and with_tests
+--- src/mapi/shared-glapi/meson.build.orig
++++ src/mapi/shared-glapi/meson.build
+@@ -36,7 +36,7 @@
+   capture : true,
+ )
+ 
+-libglapi = shared_library(
++libglapi = both_libraries(
+   'glapi',
+   [files_mapi_glapi, files_mapi_util, shared_glapi_mapi_tmp_h],
+   c_args : [
diff --git a/srcpkgs/libGL/template b/srcpkgs/libGL/template
index a88f847fca..19ded5fa3d 100644
--- a/srcpkgs/libGL/template
+++ b/srcpkgs/libGL/template
@@ -94,6 +94,7 @@ libglapi_package() {
 	short_desc="Free implementation of the GL API - shared library"
 	pkg_install() {
 		vmove "usr/lib/libglapi.so.*"
+		vmove "usr/lib/libglapi_static.a"
 	}
 }
 
diff --git a/srcpkgs/libgpg-error/template b/srcpkgs/libgpg-error/template
index 9fceb9caac..c817b8f464 100644
--- a/srcpkgs/libgpg-error/template
+++ b/srcpkgs/libgpg-error/template
@@ -3,6 +3,7 @@ pkgname=libgpg-error
 version=1.33
 revision=1
 build_style=gnu-configure
+configure_args="--enable-static"
 short_desc="Library for error values used by GnuPG component"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
 license="LGPL-2.1-or-later, GPL-2.0-or-later"
@@ -31,5 +32,6 @@ libgpg-error-devel_package() {
 		vmove usr/lib/pkgconfig
 		vmove usr/share/aclocal
 		vmove "usr/lib/*.so"
+		vmove "usr/lib/*.a"
 	}
 }
diff --git a/srcpkgs/libxcb/template b/srcpkgs/libxcb/template
index 2a4ea6fb57..4cc52eb5b5 100644
--- a/srcpkgs/libxcb/template
+++ b/srcpkgs/libxcb/template
@@ -3,7 +3,7 @@ pkgname=libxcb
 version=1.13.1
 revision=2
 build_style=gnu-configure
-configure_args="--disable-build-docs --disable-static --enable-xinput --enable-xkb"
+configure_args="--disable-build-docs --enable-static --enable-xinput --enable-xkb"
 hostmakedepends="automake libtool pkg-config xorg-util-macros xcb-proto"
 makedepends="xcb-proto libXdmcp-devel libXau-devel"
 short_desc="X protocol C-language Binding"
@@ -30,5 +30,6 @@ libxcb-devel_package() {
 		vmove usr/share
 		vmove usr/lib/pkgconfig
 		vmove "usr/lib/*.so"
+		vmove "usr/lib/*.a"
 	}
 }
diff --git a/srcpkgs/libxml2/template b/srcpkgs/libxml2/template
index e2de538978..217ae0d7ec 100644
--- a/srcpkgs/libxml2/template
+++ b/srcpkgs/libxml2/template
@@ -7,7 +7,7 @@ version=2.9.8
 revision=6
 patch_args="-Np1"
 build_style=gnu-configure
-configure_args="--disable-static --with-threads --with-history --with-icu
+configure_args="--enable-static --with-threads --with-history --with-icu
  --without-python"
 hostmakedepends="automake libtool gettext-devel pkg-config"
 makedepends="zlib-devel ncurses-devel readline-devel liblzma-devel icu-devel"
@@ -44,5 +44,6 @@ libxml2-devel_package() {
 		vmove "usr/share/man/man1/xml2-config*"
 		vmove usr/share/man/man3
 		vmove "usr/lib/*.so"
+		vmove "usr/lib/*.a"
 	}
 }
diff --git a/srcpkgs/libxslt/template b/srcpkgs/libxslt/template
index 3daf7ecccc..22d0eaf011 100644
--- a/srcpkgs/libxslt/template
+++ b/srcpkgs/libxslt/template
@@ -3,7 +3,7 @@ pkgname=libxslt
 version=1.1.32
 revision=1
 build_style=gnu-configure
-configure_args="--disable-static"
+configure_args="--enable-static"
 makedepends="python-devel libxml2-devel libxml2-python libgcrypt-devel"
 short_desc="XSLT parser library from the GNOME project"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
@@ -36,6 +36,7 @@ libxslt-devel_package() {
 		vmove usr/include
 		vmove usr/lib/pkgconfig
 		vmove "usr/lib/*.so"
+		vmove "usr/lib/*.a"
 		vmove usr/share/aclocal
 		vmove usr/share/doc/${sourcepkg}-${version}
 		vmove usr/share/man/man3
diff --git a/srcpkgs/xcb-util-keysyms/template b/srcpkgs/xcb-util-keysyms/template
index 1f274fc19f..d23ef9428c 100644
--- a/srcpkgs/xcb-util-keysyms/template
+++ b/srcpkgs/xcb-util-keysyms/template
@@ -3,7 +3,7 @@ pkgname=xcb-util-keysyms
 version=0.4.0
 revision=2
 build_style=gnu-configure
-configure_args="--disable-static"
+configure_args="--enable-static"
 hostmakedepends="pkg-config"
 makedepends="libxcb-devel xcb-util-devel"
 short_desc="Utility libraries for XC Binding - Standard X key constants and conversion to/from keycodes"
@@ -20,5 +20,6 @@ xcb-util-keysyms-devel_package() {
 		vmove usr/include
 		vmove usr/lib/pkgconfig
 		vmove "usr/lib/*.so"
+		vmove "usr/lib/*.a"
 	}
 }
